doc/source/user_guide/duplicates.rst

```diff
@@ -0,0 +1,186 @@
+.. _duplicates:
+
+****************
+Duplicate Labels
+****************
+
+:class:`Index` objects are not required to be unique; you can have duplicate row
```

Comment:
needs {{ header }} for links to work?

Reply:
No, that's just for the ipython directive.